assets/js/script.js

```diff
@@ -119,209 +102,214 @@ var randomQuestion;
 var answerBtns = document.querySelector('#answer-buttons');
-var choiceA = document.querySelector(".choice-a");
-var choiceB = document.querySelector(".choice-b");
-var choiceC = document.querySelector(".choice-c");
-var choiceD = document.querySelector(".choice-d");
-var selectedAnswer = document.querySelectorAll(".answer-btn");
+var choiceA = document.querySelector('.choice-a');
+var choiceB = document.querySelector('.choice-b');
+var choiceC = document.querySelector('.choice-c');
+var choiceD = document.querySelector('.choice-d');
+var selectedAnswer = document.querySelectorAll('.answer-btn');
 
-var contrastToggle = document.getElementById("contrast");
-var body = document.querySelector("body");
+var correctSound = document.getElementById('correct-sound');
+var wrongSound = document.getElementById('incorrect-sound');
+
+var contrastToggle = document.getElementById('contrast');
+var body = document.querySelector('body');
 
 // for day/night view toggle
-contrastToggle.addEventListener("click", function() {
-    body.classList.toggle("dark");
+contrastToggle.addEventListener('click', function () {
+  body.classList.toggle('dark');
 });
 
 // for background animation
 function ready(document) {
-    document.querySelector(".wrapper").append("<ul class='circles'><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li></ul>");
-};
+  document
+    .querySelector('.wrapper')
+    .append(
+      "<ul class='circles'><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li></ul>"
+    );
+}
 
 function startGame() {
-    pageChangeOne();
-    score = 0;
-    scoreBoard.textContent = score;
-    timerCount = 60;
-    randomQuestion = questions.sort(() => Math.random() - .5);
-    currentQuestion = 0;
-    startTimer();
-    setNextQuestion();
+  pageChangeOne();
+  score = 0;
+  scoreBoard.textContent = score;
+  timerCount = 60;
+  randomQuestion = questions.sort(() => Math.random() - 0.5);
+  currentQuestion = 0;
+  startTimer();
+  setNextQuestion();
 }
 
 function pageChangeOne() {
-    container.forEach(function(containers) {
-        if(containers.classList.contains("one")) {
-            containers.classList.add("hide");
-        }
-        if(containers.classList.contains("two")) {
-            containers.classList.remove("hide");
-        }
-    });
-};
+  container.forEach(function (containers) {
+    if (containers.classList.contains('one')) {
+      containers.classList.add('hide');
+    }
+    if (containers.classList.contains('two')) {
+      containers.classList.remove('hide');
+    }
+  });
+}
 
-function pageChangeTwo(){
-    container.forEach(function(containers) {
-        if(containers.classList.contains("two")) {
-            containers.classList.add("hide");
-        };
-        if(containers.classList.contains("three")) {
-            containers.classList.remove("hide");
-        };
-    });
-    gameEndMessage();
-};
+function pageChangeTwo() {
```

Comment:
these two funcs (pageChangeTwo, pageChangeOne) are almost exactly the same minus some variables - could you refactor this out into a generic function `pageChange(page)` or something similar that avoided having these two exist seperately?